packages/mask/src/plugins/Trader/trader/useGetTradeContext.ts

```diff
@@ -13,23 +13,28 @@ import {
     SUSHISWAP_CUSTOM_BASES,
     UNISWAP_BASE_AGAINST_TOKENS,
     UNISWAP_CUSTOM_BASES,
+    WANNASWAP_BASE_AGAINST_TOKENS,
+    WANNASWAP_CUSTOM_BASES,
+    TRISOLARIS_BASE_AGAINST_TOKENS,
+    TRISOLARIS_CUSTOM_BASES,
 } from '../constants'
 import { unreachable } from '@dimensiondev/kit'
 import { TargetChainIdContext } from './useTargetChainIdContext'
 
 export function useGetTradeContext(tradeProvider?: TradeProvider) {
     const { targetChainId: chainId } = TargetChainIdContext.useContainer()
+    const DEX_TRADE = getTraderConstants(chainId)
```

Comment:
Nice change 👍 

```ts
const DEX_TRADE = getTraderConstants(chainId)
```

Moving this line inside `useMemo` following would make the code compact, also benefit from memo.

Reply:
Ok I will change

Reply:
@UncleBill use `useTraderConstants` replace it ?